os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -84,6 +84,7 @@
 #include "ameba_soc.h"
 #include "osdep_service.h"
 #include "platform_opts_bt.h"
+#include "rtl8730e_st7701.c"
```

Comment:
looks strange. You already add it in Make.defs. I think you need a header for `rtl8730e_st7701_initialize` prototype.

Reply:
Could you check this comment?

Reply:
thanks for pointing this out, this doesn't needed. I have removed this.